xtask/Cargo.toml

```diff
@@ -10,7 +10,7 @@ test = false
 
 [dependencies]
 camino = "1.0.8"
-clap = { version = "3.2.4", features = ["derive"] }
+clap = { version = "4.0.18", features = ["derive"] }
```

Comment:
Don't we have some more oldish clap deps? In the examples?

Reply:
We have one but it's not as easy to upgrade (I think sled-state-inspector). And an indirect one through uniffi, upgrade PR at https://github.com/mozilla/uniffi-rs/pull/1383. I just did this because I noticed we can have the nicer `--help` from clap 4.0 and it was easy.